test/kubectl-asset.integ.snapshot/lambda-layer-kubectl-integ-stack.template.json

```diff
@@ -0,0 +1,380 @@
+{
+  "Resources": {
+    "KubectlLayer600207B5": {
+      "Type": "AWS::Lambda::LayerVersion",
+      "Properties": {
+        "Content": {
+          "S3Bucket": {
+            "Fn::Sub": "cdk-hnb659fds-assets-${AWS::AccountId}-${AWS::Region}"
+          },
+          "S3Key": "c6964dbf0c556ec82ce09622e99ad6f6d4e488cdaac0ef9e8492e078ec61ffed.zip"
```

Comment:
This layer version asset hash is the same as my comparable integ test in aws-cdk (which I did not publish, but can if necessary). I think that's all I need to demonstrate that the asset is the same.

Reply:
I think we should add an ntegration test to aws-cdk-lib, so that we can show the change we make in that library later is not breaking (in the happy-path scenario).